scripts/exregional_run_analysis.sh

```diff
@@ -0,0 +1,594 @@
+#   errtable = text file with obs error for conventional data (regional only)
+#   convinfo = text file with information about assimilation of conventional data
+#   bufrtable= text file ONLY needed for single obs test (oneobstest=.true.)
+#   bftab_sst= bufr table for sst ONLY needed for sst retrieval (retrieval=.true.)
+#
+#-----------------------------------------------------------------------
+
+anavinfo=${fixdir}/anavinfo_fv3sar_hrrr
+BERROR=${fixdir}/rap_berror_stats_global_RAP_tune
+SATINFO=${fixdir}/global_satinfo.txt
+CONVINFO=${fixdir}/nam_regional_convinfo_RAP.txt
+OZINFO=${fixdir}/global_ozinfo.txt
+PCPINFO=${fixdir}/global_pcpinfo.txt
+OBERROR=${fixdir}/nam_errtable.r3dv
+ATMS_BEAMWIDTH=${fixdir}/atms_beamwidth.txt
+
+# Fixed fields
+cp_vrfy "${anavinfo}" "anavinfo"
+cp_vrfy "${BERROR}"   "berror_stats"
+cp_vrfy $SATINFO  satinfo
+cp_vrfy $CONVINFO convinfo
+cp      $OZINFO   ozinfo
+cp      $PCPINFO  pcpinfo
+cp_vrfy $OBERROR  errtable
+cp_vrfy $ATMS_BEAMWIDTH atms_beamwidth.txt
+
+# Get aircraft reject list and surface uselist
+cp_vrfy ${AIRCRAFT_REJECT}/current_bad_aircraft.txt current_bad_aircraft
+
+sfcuselists=gsd_sfcobs_uselist.txt
+sfcuselists_path=${SFCOBS_USELIST}
+cp_vrfy ${sfcuselists_path}/${sfcuselists} gsd_sfcobs_uselist.txt
+cp_vrfy ${fixdir}/gsd_sfcobs_provider.txt gsd_sfcobs_provider.txt
+
+
+#-----------------------------------------------------------------------
+#
+# CRTM Spectral and Transmittance coefficients
+#
+#-----------------------------------------------------------------------
+CRTMFIX=${FIXcrtm}
+emiscoef_IRwater=${CRTMFIX}/Nalli.IRwater.EmisCoeff.bin
+emiscoef_IRice=${CRTMFIX}/NPOESS.IRice.EmisCoeff.bin
+emiscoef_IRland=${CRTMFIX}/NPOESS.IRland.EmisCoeff.bin
+emiscoef_IRsnow=${CRTMFIX}/NPOESS.IRsnow.EmisCoeff.bin
+emiscoef_VISice=${CRTMFIX}/NPOESS.VISice.EmisCoeff.bin
+emiscoef_VISland=${CRTMFIX}/NPOESS.VISland.EmisCoeff.bin
+emiscoef_VISsnow=${CRTMFIX}/NPOESS.VISsnow.EmisCoeff.bin
+emiscoef_VISwater=${CRTMFIX}/NPOESS.VISwater.EmisCoeff.bin
+emiscoef_MWwater=${CRTMFIX}/FASTEM6.MWwater.EmisCoeff.bin
+aercoef=${CRTMFIX}/AerosolCoeff.bin
+cldcoef=${CRTMFIX}/CloudCoeff.bin
+
+ln -s ${emiscoef_IRwater} Nalli.IRwater.EmisCoeff.bin
+ln -s $emiscoef_IRice ./NPOESS.IRice.EmisCoeff.bin
+ln -s $emiscoef_IRsnow ./NPOESS.IRsnow.EmisCoeff.bin
+ln -s $emiscoef_IRland ./NPOESS.IRland.EmisCoeff.bin
+ln -s $emiscoef_VISice ./NPOESS.VISice.EmisCoeff.bin
+ln -s $emiscoef_VISland ./NPOESS.VISland.EmisCoeff.bin
+ln -s $emiscoef_VISsnow ./NPOESS.VISsnow.EmisCoeff.bin
+ln -s $emiscoef_VISwater ./NPOESS.VISwater.EmisCoeff.bin
+ln -s $emiscoef_MWwater ./FASTEM6.MWwater.EmisCoeff.bin
+ln -s $aercoef  ./AerosolCoeff.bin
+ln -s $cldcoef  ./CloudCoeff.bin
+
+
+# Copy CRTM coefficient files based on entries in satinfo file
+for file in `awk '{if($1!~"!"){print $1}}' ./satinfo | sort | uniq` ;do
+#   ln_vrfy -sf -t ${CRTMFIX}/${file}.SpcCoeff.bin  .
+#   ln_vrfy -sf -t ${CRTMFIX}/${file}.TauCoeff.bin  .
```

Comment:
Please remove commented code here and below.

Reply:
I removed some comments but need to leave the part for turning on bias correction in the future.